### ISSUE TITLE
Revert Jackson bump as it looks for absent com.fasterxml.jackson:jackson-bom:2.13.2.1

### ISSUE THREAD

**Issue by nealeu** [dependency-check/dependency-check-gradle#260]
We also tried updating jackson-databind for that for our project and got the same issue you got at https://github.com/dependency-check/dependency-check-gradle/runs/5720013701?check_suite_focus=true.

The 7.0.2 release would however be great to have without the Jackson bump, as we do need the fix for #258

**Comment by Grimoren** [comment]
I second this. This is going to cause a ton of issues with all my repos.

**Comment by Grimoren** [comment]
https://github.com/jeremylong/DependencyCheck/issues/4280

**Comment by chadlwilson** [comment]
It looks like perhaps the issue is that the Gradle metadata published for `jackson-databind` is incorrect. https://github.com/FasterXML/jackson-databind/issues/3428#issuecomment-1081330160 implies this is fixed in `2.13.2.2` so should probably roll forward to that.

Overriding in buildscript classpath does work for me (at least until hitting #263 so won't help anyone else too much).
 
```
  dependencies {
    classpath 'org.owasp:dependency-check-gradle:7.0.2'
    classpath 'com.fasterxml.jackson.core:jackson-databind:2.13.2.2'
  }
```
